### PR TITLE
Modify role to install the latest 8.x version of Nessus

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request modifies the Ansible role to install the latest 8.x version of Nessus.

## 💭 Motivation and context ##

The latest 8.x version of Ansible was tested as part of the latest CyHy deployment, so we may as well use it in the COOL too.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-
 # The AWS S3 bucket containing the Nessus Debian package file
 package_bucket: "cisa-cool-third-party-production"
 
 # The name of the Nessus Debian package file
 package_file: "Nessus-{{ version }}-debian6_amd64.deb"
 
 # The version number of the Nessus Debian package file
-version: "8.7.1"
+version: "8.15.4"
```
Comment (mcdonnnj) [inline defaults/main.yml]: Shall we update to `8.15.5` while we're here since it has come out in the interim?

**Review by mcdonnnj — APPROVED** [review]
Makes sense. I had one thought for consideration.

File: defaults/main.yml (hunk 1 above)
Comment (jsf9k) [reply defaults/main.yml]: Please see commit 13df473.

**Review by dav3r — APPROVED** [review]
👍 👍

**Pull request merged**